update-po4acfg.sh

```diff
@@ -17,8 +17,9 @@ cat << EOS > $OUT_FILE
 [po4a_langs] $TARGET_LANG
 [po4a_paths] i18n/pot/\$master.pot \$lang:i18n/po/\$lang/\$master.\$lang.po
 
-[po4a_alias:myadoc] asciidoc opt:"-k 0 -M utf-8 -L utf-8"
-[po4a_alias:mytext] text opt:"-k 0 -M utf-8 -L utf-8 -o asciidoc -o neverwrap"
+# [po4a_alias:myadoc] asciidoc opt:"-k 0 -M utf-8 -L utf-8"
+# Workaround for handling asciidoc table and so on
+[po4a_alias:myadoc] text opt:"-k 0 -M utf-8 -L utf-8 -o asciidoc -o neverwrap"
```

Comment:
Unfortunately, using `asciidoc` format cannot handle table and code style of asciidoc.
Using `text` format with `-o asciidoc -o neverwrap` options looks good.